templates/pecl_extension.ini.j2

```diff
@@ -0,0 +1,2 @@
+; {{ ansible_managed| comment('plain', decoration=';') }}
```

Comment:
If you add the ansible_managed function you can remove the semicolon in the column1